src/IRac.cpp

```diff
@@ -3101,6 +3105,11 @@ int16_t IRac::strToModel(const char *str, const int16_t def) {
     return whirlpool_ac_remote_model_t::DG11J13A;
   } else if (!strcasecmp(str, "DG11J191")) {
     return whirlpool_ac_remote_model_t::DG11J191;
+  // Daikin models
+  } else if (!strcasecmp(str, "ARC433XX")) {
+    return daikin_ac_remote_model_t::ARC433XX;
+  } else if (!strcasecmp(str, "ARC484A4")) {
+    return daikin_ac_remote_model_t::ARC484A4;
```

Comment:
Any further thoughts on the global models idea?

Reply:
Soooo ... I played around with this the other day actually. I tried putting this in `PROGMEM` (via `F()`) when I was working on #1551.
I was quickly reminded why it is the way it is, and that I had tried this before. :-(

The problem/issue is `strcasecmp()` doesn't like Arduino's Flashhelper. It complains bitterly about it.
There probably is an Arduino `strcasecmp_f()` (I just looked, it doesn't seem to exist) or something that works, but it's not in normal C++ even if it was, thus would cause the Unit Tests (as is) to break. I'd have to do something fancy like make a framework dependant macro to swap between the two. Or write a stub-ish function.

So , yes. I've thought about it a little, and I put it back in the "too hard for right now" basket. :-/